indico/modules/events/abstracts/templates/management/abstract_list.html

```diff
@@ -0,0 +1,125 @@
+                <a class="i-button icon-disable js-requires-selected-row js-dialog-action danger">
+                    {% trans %}Reject{% endtrans %}
+                </a>
+                {# TODO: Add icon for merge #}
+                <a class="i-button js-requires-selected-row js-dialog-action"
+                   {# TODO: data-href="{{ url_for('.merge_abstracts', event) }}" #}>
+                    {% trans %}Merge{% endtrans %}
+                </a>
+            </div>
+            <div class="group left">
+                <button class="i-button icon-users js-enable-if-checked disabled"
+                        data-title="{% trans %}Author list{% endtrans %}"
+                        {# TODO: data-href="{{ url_for('.person_list', event) }}" #}
+                        data-params-selector="#abstract-list input[name=abstract_id]:checked"
+                        data-method="POST"
+                        data-dialog-classes="event-persons-dialog"
+                        data-ajax-dialog>
+                    {%- trans %}Author list{% endtrans -%}
+                </button>
+            </div>
+            <div class="group left">
+                <a href="#" class="i-button icon-attachment js-enable-if-checked js-submit-form disabled"
+                   {# TODO: data-href="{{ url_for('.download_attachments', event) }}" #}>
+                    {%- trans %}Download attachments{%- endtrans %}
+                </a>
+            </div>
+            <div class="group left">
+                <a class="i-button arrow button js-requires-selected-row disabled" data-toggle="dropdown">
+                    {%- trans %}Export{% endtrans -%}
+                </a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="#"
+                           class="icon-file-pdf js-requires-selected-row disabled js-submit-list-form"
+                           {# TODO: data-href="{{ url_for('.abstracts_pdf_export', event) }}" #}>
+                            {%- trans %}PDF{% endtrans -%}
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#"
+                           class="icon-file-spreadsheet js-requires-selected-row disabled js-submit-list-form"
+                           {# data-href="{{ url_for('.abstracts_csv_export', event) }}" #}>CSV</a>
+                    </li>
+                    <li>
+                        <a href="#"
+                           class="icon-file-excel js-requires-selected-row disabled js-submit-list-form"
+                           {# TODO: data-href="{{ url_for('.abstracts_excel_export', event) }}" #}>XLSX (Excel)</a>
+                    </li>
+                    <li>
+                        <a href="#"
+                           class="icon-file-xml js-requires-selected-row disabled js-submit-list-form"
+                           {# TODO: data-href="{{ url_for('.abstracts_xml_export', event) }}" #}>XML</a>
```

Comment:
Maybe one of the two brace icons would fit?

![icons](https://cloud.githubusercontent.com/assets/179599/17846484/10297eea-6849-11e6-9469-ff706a10410f.png)